ocs_ci/templates/workloads/fio/config_s3.fio

```diff
@@ -32,3 +32,24 @@ size=64k
 # bs=4k
 # size=64k
 # io_size=4k
+
+[job1]
+iodepth=4
+rw=randrw
+bs=32k
+size=64m
+numjobs=4
+
+[job2]
+iodepth=16
+rw=randrw
+bs=64k
+size=512m
+numjobs=4
+
+[job3]
+iodepth=32
+rw=randrw
+bs=128k
+size=1024m
+numjobs=4
```

Comment:
What will happen to all the other code that uses `mcg_job_factory`? will default invocation will ignore all these new jobs?

Reply:
if you don't have options specify in the test case, mcg_job_factory will pick up default fio config which doesn't have enough load.  Default config is set: https://github.com/red-hat-storage/ocs-ci/blob/8d44436d54ab29e581a16ce0783a62a2a8601811/ocs_ci/templates/workloads/fio/config_s3.fio#L22

Reply:
This is actually the default config file. This way all tests that use `mcg_job_factory` would use this configuration. I suggest to add these options to test in `tests/manage/mcg/test_endpoint_autoscale.py` like:
```
options = {
    'create': [('name', 'job1'), ('name', 'job2'), ('name', 'job3'), ('runtime', '900')],
    'job1':[('iodepth', '4'), ('rw', 'randrw'), ('bs', '32k'), ('size', '64m'), ('numjobs', '4')],
    'job2':[('iodepth', '16'), ('rw', 'randrw'), ('bs', '64k'), ('size', '512m'), ('numjobs', '4')],
    'job3':[('iodepth', '32'), ('rw', 'randrw'), ('bs', '128k'), ('size', '1024m'), ('numjobs', '4')]
}
job = mcg_job_factory(custom_options=options)
```

Reply:
I prefer @fbalak suggestion/solution as it was my original recommendation to do it that way

Reply:
@nb-ohad I will sync up with @fbalak on this.

Reply:
Fixed.